config_webrx.py

```diff
@@ -137,6 +141,53 @@
             },
         },
     },
+    "perseussdr": {
+        "name": "Microtelecom Perseus HF receiver",
+        "type": "perseussdr",
+        "ppm": 0,
+        "profiles": {
+            "40m": {
+                "name": "40m",
+                "center_freq": 7150000,
+                "rf_gain": 0,
+                "samp_rate": 500000,
+                "start_freq": 7050000,
+                "start_mod": "lsb",
+            },
+            "20m": {
+                "name": "20m",
+                "center_freq": 14150000,
+                "rf_gain": 0,
+                "samp_rate": 500000,
+                "start_freq": 14070000,
+                "start_mod": "usb",
+            },
+            "30m": {
+                "name": "30m",
+                "center_freq": 10125000,
+                "rf_gain": 0,
+                "samp_rate": 192000,
+                "start_freq": 10142000,
+                "start_mod": "usb",
+            },
+            "80m": {
+                "name": "80m",
+                "center_freq": 3650000,
+                "rf_gain": 0,
+                "samp_rate": 500000,
+                "start_freq": 3570000,
+                "start_mod": "usb",
+            },
+            "49m": {
+                "name": "49m Broadcast",
+                "center_freq": 6000000,
+                "rf_gain": 0,
+                "samp_rate": 500000,
+                "start_freq": 6070000,
+                "start_mod": "am",
+            },
+        },
+    },
```

Comment:
I'd like to keep it out of the default configuration. I'd say the wiki is a better place to keep all the different configuration samples.

Reply:
Exactly where have I to put  in the Wiki ?

Reply:
i'd say just open up a new page. I started off with separate pages for airspies. I'll need to build some kind of listing / index at some point.

Reply:
Ahh, now I understand! I was confusing the old Wiki in the Andras repo, with the new one in yours.